### DISCUSSION TITLE
C6800-16P10G-XL module-type

### DISCUSSION THREAD

**Discussion by micha11938** [netbox-community/devicetype-library#2090] [Show and tell]
Asked for a module template for Cisco Catalyst 6807-XL.  I can't find my request anymore.
Anyway, I just made one and imported it.

[C6807-16P10G-XL.zip](https://github.com/netbox-community/devicetype-library/files/14996162/C6807-16P10G-XL.zip)

**Comment by danner26** [comment]
Please create a PR against this repo, we would be happy to review a PR and merge it when it is ready
